apps/emqx_bridge_kafka/src/emqx_bridge_kafka_action_info.erl

```diff
@@ -20,3 +22,19 @@ action_type_name() -> kafka_producer.
 connector_type_name() -> kafka_producer.
 
 schema_module() -> emqx_bridge_kafka.
+
+action_to_bridge_v1_fixup(Config) ->
```

Comment:
Nit: since it's currently the same, this callback and its inverse could delegate to Kafka's module to avoid repetition.

Reply:
Yes, we could avoid repetition of the this code in the azure bridge. This is what you meant right?